game/src/sandbox/dashboards/parking_overhead.rs

```diff
@@ -1,20 +1,104 @@
 use geom::Duration;
 use sim::{TripEndpoint, TripID, TripPhaseType};
 use widgetry::table::{Col, Filter, Table};
-use widgetry::{EventCtx, Filler, Line, Panel, State, Text, Toggle, Widget};
+use widgetry::{EventCtx, Filler, GfxCtx, Line, Outcome, Panel, State, Text, Toggle, Widget};
+
+type Transition = widgetry::Transition<App>;
 
 use crate::app::App;
```

Comment:
You can pull in `crate::app::Transition` here instead of the typedef above

Reply:
done!